test/go-tests/sequencetrigger_test.go

```diff
@@ -0,0 +1,289 @@
+package go_tests
+
+import (
+	"fmt"
+	keptncommon "github.com/keptn/go-utils/pkg/lib/keptn"
+	keptnv2 "github.com/keptn/go-utils/pkg/lib/v0_2_0"
+	"github.com/stretchr/testify/require"
+	"net/http"
+	"os"
+	"testing"
+	"time"
+)
+
+const sequenceTriggerShipyard = `apiVersion: "spec.keptn.sh/0.2.0"
```

Comment:
I think this is the wrong spec version, as spec 0.2.0 does not support the selector to match a task